spec/section/fields.md

```diff
@@ -1,5 +1,9 @@
 ## Fields {#fields}
 
+<aside class="issue">
+Are we sure that fields are expression maps? I am not really convinced about that. We do not want to allow for things such as `rr:template` on fields, right?
```

Comment:
Although rml:template and rml:constant would not be very useful, we have not excluded this during the Santiago meeting. I do see the need/benefit of allowing function values. Therefore, I still support that fields are defined as expression maps

Reply:
Hi Els, I am still unconvinced. Could you make an example where this makes sense? I like to keep stuff related to "sources" conceptually distinct from stuff related to the "target" (i.e., the triples we intend to generate). In my view, the notion of LogicalView belongs to the source level.

Reply:
1. Data transformations (as a result of a function > function values) are sometimes needed to join to data sources on certain fields. E.g. one source contains a field with data in lower case, and the other source contains a field with the same data in upper case. You first need to transform on field before you can used that field to in the join condition to join the two sources. 
2. It can be better to define a data transformation at the source level - field level, if you need the same transformation afterwards in several places in the mapping. See also (Fun Mapping)[https://arxiv.org/abs/2008.13482]

Reply:
Sure, please go ahead.